src/ontology/ro-edit.owl

```diff
@@ -6542,6 +6545,30 @@ AnnotationAssertion(oboInOwl:creation_date obo:RO_0017001 "2021-11-08T12:00:00Z"
 AnnotationAssertion(oboInOwl:hasBroadSynonym obo:RO_0017001 "utilizes")
 AnnotationAssertion(rdfs:label obo:RO_0017001 "device utilizes material"@en)
 
+# Object Property: obo:RO_0017008 (owns)
+
+AnnotationAssertion(obo:IAO_0000112 obo:RO_0017008 "biobank organization owns a specimen")
+AnnotationAssertion(obo:IAO_0000112 obo:RO_0017008 "hospital owns a laboratory facility")
+AnnotationAssertion(Annotation(dc:source "OBIB:0000735") obo:IAO_0000115 obo:RO_0017008 "A primitive relation that holds between entities x and y in which y is at x’s full disposal.")
+AnnotationAssertion(dc:contributor obo:RO_0017008 <https://orcid.org/0000-0001-9625-1899>)
+AnnotationAssertion(dc:contributor obo:RO_0017008 <https://orcid.org/0000-0003-1834-3856>)
+AnnotationAssertion(dc:contributor obo:RO_0017008 <https://orcid.org/0000-0003-1909-9269>)
+AnnotationAssertion(dc:description obo:RO_0017008 "This primitive relation is the foundation of the owner’s right to have the owned entity at his/her full disposal.")
+AnnotationAssertion(oboInOwl:creation_date obo:RO_0017008 "2023-03-29T16:57:03Z"^^xsd:dateTime)
+AnnotationAssertion(rdfs:label obo:RO_0017008 "owns"@en)
+InverseObjectProperties(obo:RO_0017008 obo:RO_0017009)
+
+# Object Property: obo:RO_0017009 (is owned by)
+
+AnnotationAssertion(obo:IAO_0000112 obo:RO_0017009 "freezer is owned by a biobank organization")
+AnnotationAssertion(Annotation(dc:source "OBIB:0000735") obo:IAO_0000115 obo:RO_0017009 "Inverse of the owns relation.")
+AnnotationAssertion(dc:contributor obo:RO_0017009 <https://orcid.org/0000-0001-9625-1899>)
+AnnotationAssertion(dc:contributor obo:RO_0017009 <https://orcid.org/0000-0003-1834-3856>)
+AnnotationAssertion(dc:contributor obo:RO_0017009 <https://orcid.org/0000-0003-1909-9269>)
```

Comment:
Need to change the annotation to `terms:contributor`

Reply:
@wdduncan Yes, looks good, thanks!

Reply:
@CDowland please see @anitacaron comment. Can you fix?

> Need to change the annotation to terms:contributor